lib/rubocop/cop/vendor/disallow_after_commit.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Vendor
+      class DisallowAfterCommit < Base
+        extend AutoCorrector
+
+        NO_METH_MSG = "You should not use `%<old>s` %<desc>s"
```

Comment:
nit: I prefer rules that state imperatives rather than point at the individual "you". It is a rule after all. 

```suggestion
        NO_METH_MSG = "Do not not use `%<old>s` %<desc>s"
```